docs/CHANGELOG.adoc

```diff
@@ -0,0 +1,58 @@
+= Changelog
+All notable changes to this project will be documented in this file since 
+July 11 of 2019 (2019-07-11).
+
+The format is based on https://keepachangelog.com/en/1.0.0/[Keep a Changelog],
+and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Versioning].
+
+== [Unreleased]
+
+== [0.2.0] - 2019-07-14
+=== Added
+* Locking function for the die in case you want to not accidentally modify the die 
+(i.e. adding/removing items, editing items) or something.
+
+* Disabling dice now prevents you from modifying anything in the die 
+(it's not a bug, it's a feature, I swear...). It basically functions as a lock 
+function except it doesn't count towards any of the app's operations as if it 
+was deleted or never existed in the first place.
+
+* Set a maximum length set for die name, die items, and idea set names. Here's the 
+following specifications:
+** For die and idea set names, *its length must be less than 64 characters*.
+** For die items, *its length must be less than 32 characters*.
+** There's no format for the required characters. You can put all of the exclamation marks, 
+emojis, and semicolons all you want.
+
+* 'Undo' function for previously deleted dice (you can only retrieve up to 20 recently 
+deleted dice at any given time). Take note that you can only hold it in the current session. 
+Changing pages or reloading will lose the deleted dice stack.
+
+* New keyboard shortcuts to the main app page
+** `Shift + N` for a new die
+** `Ctrl + G` for generating an idea set
+** `Ctrl + D` for clearing the generated output
+** `Ctrl + S` for saving the output in the idea set database
+** `Ctrl + I` for importing dice set file
+** `Ctrl + E` for exporting the current dice set
+** `Ctrl + Delete` for deleting the current die shown in the screen
```

Comment:
This has been implemented as 'Shift + Delete', not 'Ctrl + Delete'?

 Line 22 at src/components/DieComponent.vue

Reply:
`Shift + Delete` is for deleting the currently selected die item.

`Ctrl + Delete` is for deleting the die (including the die items).

What you're looking for can be found at `src/layout/DieMenu.vue` starting at line 288.

P.S. Seems like I forgot to write about it in the changelog. Sorry about that.